bin/verify-sql-formatting.py

```diff
@@ -0,0 +1,104 @@
+import argparse
+import difflib
+import os
+import re
+import shutil
+import sqlparse
+
+
+def print_without_common_lines(original, parsed):
+    d = difflib.Differ()
+    diff = d.compare(
+        original.splitlines(keepends=True), parsed.splitlines(keepends=True)
+    )
+    buffer = []
+    counter = 1
+    for line in diff:
```

Comment:
Rather than building your own output, do you just want `difflib.ndiff()`?

Reply:
I tested it a bit, but it did not really do what I hoped to see. Maybe my config was off?